spec/calc_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+require 'calc'
+
+RSpec.describe Calc do
+  describe '# sum' do
+    it 'retunrs sum of 2 numbers' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.